Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.18-bullseye AS builder
+
+ADD . /workspace
+WORKDIR /workspace
+RUN make build
+
+FROM golang:1.18-bullseye
```

Comment:
Do you want to include all the golang tools in the final image?

Either way, LGTM!

Reply:
there's some tools in the agoric-sdk (e.g. jq) that I need for some runtime stuff as well.  If we need a slim image I'm happy to create one too